src/runtime/pythonexception.cs

```diff
@@ -77,7 +77,7 @@ public PythonException()
             }
             _finalized = true;
             Finalizer.Instance.AddFinalizedObject(ref _pyType);
-            Finalizer.Instance.AddFinalizedObject(ref _pyValue);
+            // Finalizer.Instance.AddFinalizedObject(ref _pyValue);
```

Comment:
@BadSingleton here.

Reply:
Thanks! I'll look at that, I haven't found a definite culprit yet.